### PR TITLE
Kucero signs kubelet server csr

### PR DESCRIPTION
**Pull request by jenting**
# Describe your changes

Adding kubelet server CSR to the certificate list and when will kubelet server CSR send out.

# Related Issues / Projects

Relates to: https://github.com/SUSE/avant-garde/issues/1958
fixes #1002

### CODE REVIEW

**Comment by jenting** [comment]
Waits PR https://github.com/SUSE/skuba/pull/1365 to be merged

File: adoc/admin-security-certificates.adoc [hunk 1]
```diff
@@ -97,6 +97,11 @@ The control plane certificates stored in the {kube} cluster on control plane nod
 |/var/lib/kubeket/pki/kubelet-client-current.pem
 |Client
 
+|system:node:<nodeName>
+|kubelet-ca
+|/var/lib/kubeket/pki/kubelet-server-current.pem
```
Comment (innobead) [inline adoc/admin-security-certificates.adoc:102]:
```suggestion
|/var/lib/kubelet/pki/kubelet-server-current.pem
```

**Review by innobead — APPROVED** [review]
LGTM, but some typos needed to fix.

File: adoc/admin-security-certificates.adoc [hunk 2]
```diff
@@ -553,16 +558,17 @@ sed -i "s|oidc-ca-file=.*|oidc-ca-file=/etc/kubernetes/pki/oidc-ca.crt|" /etc/ku
...
 
-.kubelet client certificate
+.kubelet client or server certificate
 [NOTE]
 ====
-During the kubelet client certificate signing flow, kubelet sends the kubelet client CSR into the {kube} cluster. The `kube-controller-manager` signs the kubelet client CSR with the Kubernetes CA cert/key pair. The `kucero` signs the kubelet server CSR with the kubelet CA cert/key pair.
+During the kubelet client or server certificate signing flow, kubelet sends the kubelet client or server CSR into the {kube} cluster. The `kube-controller-manager` signs the kubelet client CSR with the Kubernetes CA cert/key pair, the `kucero` signs the kubelet server CSR with the kubelet CA cert/key pair.
 
-The kubelet client certificate renews automatically at approximately 70%-90% of the total lifetime of the certificate, the kubelet would use new client certificates without downtime.
+The kubelet client or server certificate renews automatically at approximately 70%-90% of the total lifetime of the certificate, the kubelet would use new client orserver certificates without downtime.
```
Comment (c3y1huang) [inline adoc/admin-security-certificates.adoc:568]:
```suggestion
The kubelet client or server certificate renews automatically at approximately 70%-90% of the total lifetime of the certificate, the kubelet would use new client or server certificates without downtime.
```

**Comment by jenting** [comment]
Waits skuba PR https://github.com/SUSE/skuba/pull/1375 to be merged.

**Review by cclhsu — APPROVED** [review]

**Pull request merged**